omaha_server/omaha/request.xsd

```diff
@@ -117,13 +122,17 @@
             <xs:attribute name="time_since_download_start_ms" use="optional" type="xs:integer"/>
             <xs:attribute name="nextversion" use="optional" type="xs:integer"/>
             <xs:attribute name="previousversion" use="optional" type="xs:integer"/>
+            <xs:attribute name="url" use="optional" type="xs:string"/>
+            <xs:attribute name="is_bundled" use="optional" type="xs:string"/>
         </xs:complexType>
     </xs:element>
     <xs:element name='ping'>
         <xs:complexType>
             <xs:attribute name='active' use='optional' type='xs:integer' default="0"/>
             <xs:attribute name='r' use='optional' type='xs:integer' default="0"/>
             <xs:attribute name='a' use='optional' type='xs:integer' default="0"/>
+            <xs:attribute name='rd' use='optional' type='xs:string' default="0"/>
```

Comment:
Default value is -2 as mentioned it protocol documentation.
You can add also

```
 <xs:attribute name='ad' use='optional' type='xs:integer' default="-2"/>
```
rule.